avaframe/AvaFrame
# Add release test

* Add pytest for getReleaseArea
* Change creating domain extent vectors, fix bug if x or y should be symmetrical around zero
* set max height of helix to total fall height
* update release areas and geometries for test cases 
* add release area name to shapefiles
